querybook/webapp/components/QueryExecution/QueryExecutionDuration.tsx

```diff
@@ -0,0 +1,52 @@
+import moment from 'moment';
+import * as React from 'react';
+
+import { IQueryExecution, QueryExecutionStatus } from 'const/queryExecution';
+import { formatDuration } from 'lib/utils/datetime';
+
+import { UserName } from 'components/UserBadge/UserName';
+import { useQueryExecutionReduxState } from './QueryExecution';
+import { StyledText } from 'ui/StyledText/StyledText';
+import { TaskStatusIcon } from 'components/Task/TaskStatusIcon';
+import { TaskRunStatus } from 'const/schedule';
+
+interface IProps {
+    queryExecution: IQueryExecution;
+}
+
+export const QueryExecutionDuration: React.FunctionComponent<IProps> = ({
+    queryExecution,
+}) => {
+    const { statementExecutions } = useQueryExecutionReduxState(
```

Comment:
you don't need statementExecution completed at, you can use queryExecution.completed_at